src/Widgets/CalendarButton.vala

```diff
@@ -45,6 +45,7 @@ public class Maya.View.Widgets.CalendarButton : Gtk.MenuButton {
 
         var button_grid = new Gtk.Grid ();
         button_grid.column_spacing = 6;
+        button_grid.valign = Gtk.Align.CENTER;
```

Comment:
This is not strictly related to this PR, but simply vertically centers the text within the button, leaving it more aesthetically pleasing.

Reply:
I don't see any difference when I switch between `CENTER` and `FILL`, but this makes enough sense to me.